### PR TITLE
Support setting 'select' in a query profile

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: vespajlib/src/test/java/com/yahoo/tensor/MixedTensorTestCase.java [hunk 1]
```diff
@@ -159,4 +164,25 @@ public void testTwoMappedTwoIndexedBuilding() {
                 tensor.toString());
     }
 
+    @Test
+    public void testSplitIntoDense() {
```
Comment (baldersheim) [inline vespajlib/src/test/java/com/yahoo/tensor/MixedTensorTestCase.java:168]: Unrelated I assume.
Looks like documentation for the code you gave to @andreer  the other day.

**Review by baldersheim — APPROVED** [review]
Functionality lost between 7 or 8, or later on ?

File: vespajlib/src/test/java/com/yahoo/tensor/MixedTensorTestCase.java (hunk 1 above)
Comment (bratseth) [reply vespajlib/src/test/java/com/yahoo/tensor/MixedTensorTestCase.java:168]: Yes, decided it was worth keeping + lazy ...

**Comment by bratseth** [comment]
I think later than 7 -> 8 but not sure.

**Pull request merged**